gui/wxpython/image2target/ii2t_manager.py

```diff
@@ -895,7 +895,7 @@ def OnSrcSelection(self, event):
             if p.returncode == 0:
                 print("returncode = ", str(p.returncode))
                 self.parent.Map.region = self.parent.Map.GetRegion()
-        except:
```

Comment:
This is the same chunk of code we already saw, it's just copied over. You can remove the entire block.